server/remote/msgraph/get_user_mailbox_settings.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package msgraph
+
+import (
+	"net/http"
+
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/remote"
+)
+
+func (c *client) GetUserMailboxSettings(remoteUserID string) (*remote.MailboxSettings, error) {
+	var u string
+	if remoteUserID == "me" {
```

Comment:
Do we need this logic? 0/5 with the exception of the first contact(?) are we ever going to use /me? We always have the UserID available to us, do we not?